README.md

```diff
@@ -103,6 +107,14 @@ the project:
 ``
 
 and then run the project with IOS emulator.
+---
+
+Some version of Gradle create unnecessary files that may result in errors while building the app. Gradle 2.14.1 has proven to work without any issues.
+If errors persist, make sure gradle-wrapper.properties is using the right version of Gradle.
+---
+
+The Android platform requires Android 6.0, make sure you have installed the necessary API levels in the SDK manager.
```

Comment:
Specify the API level as well?
e.g. Android 6.0 -> Android 6.0 (API 23?)
or the necessary API levels (API 23)